docs/source/installation.rst

```diff
@@ -71,15 +71,13 @@ On Unix system, one may have to modify udev rules to allow non-root access to
 the device you are trying to connect to. The following tutorial describes how
 to do it http://ask.xmodulo.com/change-usb-device-permission-linux.html.
 
-On Windows, you may have to uninstall the USBTMC specific driver installed by
-Windows and re-install a generic driver.
+On Windows, you may have to uninstall the USBTMC-specific driver installed by
+Windows and re-install a generic driver. Please check `libusb's guide`_ for more
+details, but installing a ``WinUSB`` driver with Zadig_ should be a good start.
 
 Note that on Windows, devices that are already open cannot be detected and will
 not be returned by ``ResourceManager.list_resources``.
 
-Another useful reference for how to configure your system is h
```

Comment:
Is this not relevant anymore ?

Reply:
I don't see much there in terms of Windows support beyond a (broken) link to libusb wiki, and their paragraph on INF files actually confused me more, so I think it's ok to replace the link wholesale.